src/connectors/aws_collect.py

```diff
@@ -88,803 +88,866 @@
-                    'SnsTopicARN': 'sns_topic_arn',
-                    'IncludeGlobalServiceEvents': 'include_global_service_events',
-                    'IsMultiRegionTrail': 'is_multi_region_trail',
-                    'HomeRegion': 'home_region',
-                    'TrailARN': 'trail_arn',
-                    'LogFileValidationEnabled': 'log_file_validation_enabled',
-                    'CloudWatchLogsLogGroupArn': 'cloud_watch_logs_log_group_arn',
-                    'CloudWatchLogsRoleArn': 'cloud_watch_logs_role_arn',
-                    'KmsKeyId': 'kms_key_id',
-                    'HasCustomEventSelectors': 'has_custom_event_selectors',
-                    'HasInsightSelectors': 'has_insight_selectors',
-                    'IsOrganizationTrail': 'is_organization_trail',
-                }
-            ]
-        },
-        'children': [
-            {'method': 'cloudtrail.get_trail_status', 'args': {'Name': 'trail_arn'}},
-            {
-                'method': 'cloudtrail.get_event_selectors',
-                'args': {'TrailName': 'trail_arn'},
-            },
-        ],
-    },
-    'cloudtrail.get_trail_status': {
-        'params': {'Name': 'trail_arn'},
-        'response': {
-            'IsLogging': 'is_logging',
-            'LatestDeliveryError': 'latest_delivery_error',
-            'LatestNotificationError': 'latest_notification_error',
-            'LatestDeliveryTime': 'latest_delivery_time',
-            'LatestNotificationTime': 'latest_notification_time',
-            'StartLoggingTime': 'start_logging_time',
-            'StopLoggingTime': 'stop_logging_time',
-            'LatestCloudWatchLogsDeliveryError': 'latest_cloud_watch_logs_delivery_error',
-            'LatestCloudWatchLogsDeliveryTime': 'latest_cloud_watch_logs_delivery_time',
-            'LatestDigestDeliveryTime': 'latest_digest_delivery_time',
-            'LatestDigestDeliveryError': 'latest_digest_delivery_error',
-            'LatestDeliveryAttemptTime': 'latest_delivery_attempt_time',
-            'LatestNotificationAttemptTime': 'latest_notification_attempt_time',
-            'LatestNotificationAttemptSucceeded': 'latest_notification_attempt_succeeded',
-            'LatestDeliveryAttemptSucceeded': 'latest_delivery_attempt_succeeded',
-            'TimeLoggingStarted': 'time_logging_started',
-            'TimeLoggingStopped': 'time_logging_stopped',
-        },
-    },
-    'cloudtrail.get_event_selectors': {
-        'response': {
-            'TrailARN': 'trail_arn',
-            'EventSelectors': [
-                {
-                    'ReadWriteType': 'read_write_type',
-                    'IncludeManagementEvents': 'include_management_events',
-                    'DataResources': 'data_resources',
-                    'ExcludeManagementEventSources': 'exclude_management_event_sources',
-                }
-            ],
-        }
+    # 'organizations.list_accounts': {
```

Comment:
my guess is that it's not working because you commented these out. it needs to be able to know what accounts there are before running anything else